src/schema/objects/metadata.yaml

```
@@ -2825,6 +3023,55 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Screen distance in cm
+    (for example `60` for a screen distance of 60 cm), if no screen use `n/a`.
+  anyOf:
+    - type: number
+      unit: cm
+    - type: string
+      enum:
+        - n/a
+ScreenRefreshRate:
+  name: ScreenRefreshRate
+  display_name: Screen Refresh Rate
+  description: |
+    Refresh rate of the screen (when used), in Hertz
+    (for example, `60` for 60 Hz).
```

```suggestion
    Refresh rate of the screen (when used), in Hertz.
```

I don't think you need an example when it's just a number without any kind of conversion/modification.

---

OK